courses/models.py

```diff
@@ -427,6 +426,18 @@ def save(
         )
 
 
+class BlockedCountry(TimestampedModel):
+    """Represents a country that is blocked from enrollment for a course"""
+
+    course = models.ForeignKey(
+        Course, on_delete=models.CASCADE, related_name="blocked_countries"
+    )
+    country = CountryField()
```

Comment:
Although it doesn't affect the logic we added to check the blocked country upon enrollment but we can still create multiple entries with the same data for this Model. My opinion is that we make them `unique_together`, not allowing multiple entries.

Reply:
Yup, I agree